turbinia/output_manager.py

```diff
@@ -495,3 +496,49 @@ def copy_from(self, source_path):
       log.error(message)
       raise TurbiniaException(message)
     return destination_path
+
+  def copy_from_gcs(self, saved_paths):
+    """Copies output file from the managed location to the local output dir.
+
+    Args:
+      saved_paths (list): A list of saved paths. List includes GCS and local
+      output file paths.
+
+    Raises:
+      TurbiniaException: If file retrieval fails.
+    """
+    bucket = self.client.get_bucket(self.bucket)
+
+    for path in saved_paths:
+      if 'gs://' in path:
+        gcs_path = self._parse_gcs_path(path)[1]
+
+        try:
+          # Reconstruct the same file structure as GCS on the output dir
+          path_split = gcs_path.split("/")
+          directory = "/".join(path_split[0:-1])
```

Comment:
Minor thing, but it's more idiomatic to say:
```suggestion
          directory = os.path.join(path_split[0:-1])
```